### PR TITLE
Update based on voxpupuli/modulesync_config

### PR DESCRIPTION
**Pull request by dhoppe**
Please merge.

### CODE REVIEW

File: metadata.json [hunk 1]
```diff
@@ -1,12 +1,12 @@
 {
   "name": "puppet-mcollective",
   "version": "2.1.3",
-  "author": "puppetcommunity",
+  "author": "puppet",
```
Comment (tuxmea) [inline metadata.json]: Author puppet? Not voxpupuli?

**Comment by dhoppe** [comment]
@daenney @tuxmea I am using ```puppet``` because it is the namespace of the Puppet Forge, but maybe someone should make a decision if we are using ```puppet``` or ```voxpupuli``` as author.

**Comment by daenney** [comment]
We're using `puppet` since that's the user we have and renaming things on the Forge is a nightmare because it has no concept of redirects.

File: metadata.json (hunk 1 above)
Comment (igalic) [inline metadata.json]: naming is hard :|

**Pull request merged**